components/resources/ResourceTab.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import {
+  CategoryCard,
+  CategoryContainer,
+  CategoryIcon,
+  HeadingContainer,
+  HeadingText
+} from '../../styled/resources';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Colors from '../../assets/Colors';
+
+class ResourceTab extends React.Component {
+  render() {
+    return (
+      <CategoryCard>
+        <CategoryContainer>
+          <CategoryIcon>
+            <FontAwesome5
+              name={this.props.icon}
+              size={24}
+              color={Colors.darkerGreen}
+            />
+          </CategoryIcon>
+        </CategoryContainer>
+        <HeadingContainer>
```

Comment:
You don't need either `CategoryContainer` or `HeadingContainer` -- see other comments.